bedrock/firefox/helpers.py

```diff
@@ -145,14 +152,15 @@ def download_firefox(ctx, channel='release', small=False, icon=True,
     data = {
         'locale_name': locale_name,
         'version': version,
-        'product': 'firefox-android' if platform == 'android' else 'firefox',
+        'product': 'firefox-%s' % platform,
```

Comment:
the possible values are controlled by product-details, so other values could be introduced (either intentionally or not) and we need to identify what would happen in that case.